client_mining_p/miner.py

```diff
@@ -1,6 +1,6 @@
 import hashlib
 import requests
-
+import time
```

Comment:
Nice to do stretch. Make your commit as that so it easier to look for code that does relevant to commit message